ts/kpt-functions/tsconfig.json

```diff
@@ -19,8 +20,8 @@
     "rootDir": ".",
     "sourceMap": true,
     "strict": true,
-    "target": "es2017"
+    "target": "es2018"
   },
   "exclude": ["node_modules", "dist"],
-  "include": ["index.ts", "src/**/*.ts"]
```

Comment:
Make this consistent for all tsconfig files. I prefer src/**/*

Reply:
Done already